frontend/public/locales/en/translation.json

```diff
@@ -3194,6 +3204,7 @@
   "You can include any combination of filters and keywords to make your search more specific.": "You can include any combination of filters and keywords to make your search more specific.",
   "You cannot create new clusters from deleted credentials. Clusters that you previously created will not be affected.": "You cannot create new clusters from deleted credentials. Clusters that you previously created will not be affected.",
   "You do not have any recommendations yet.": "You do not have any recommendations yet.",
+  "You don't have any {{kindHead}} policies": "You don't have any {{kindHead}} policies",
   "You don't have any applications": "You don't have any applications",
   "You don't have any channels": "You don't have any channels",
   "You don't have any clusters reporting policy status": "You don't have any clusters reporting policy status",
```

Comment:
Thanks for the suggestion here @dockerymick but since this PR is just adding/modifying the "You don't have any {{kindHead}} policies" sentence, let's only update that one. The others are out of scope and would require a Jira issue as some of these sentences are used outside of the GRC portion of the UI.